Decorator/trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    if @nameable.correct_name.length > 10
+      "#{@nameable.correct_name[0..9]}.."
```

Comment:
- [ ] Great job trimming the name, but I notice you added `..` at the end of the name. Kindly remove this please, the name should only be trimmed.

Reply:
Thanks for your review!
Microverse CR never stops surprising me, really blocked my approval for adding ".." to the end of the string.

Reply:
> Thanks for your review! Microverse CR never stops surprising me, really blocked my approval for adding ".." to the end of the string.

I apologize, sir, Just doing my job.

Reply:
Microverse tells the Code Reviewver to pay attention to project requirements, any other information you want to share should be marked as OPTIONAL, yes that badge that you probably received a lot during your journey as Micronaut but never used as Code Reviewver.

Reply:
I apologize sir. Just doing my job. :100: :)

Reply:
> Microverse tells Code Reviewvers to pay attention to project requirements, any other information you want to share should be marked as OPTIONAL, yes that badge that you probably received a lot during your journey as Micronaut but never used as Code Reviewver.

![image](https://user-images.githubusercontent.com/94551484/226143289-5f14c5a4-2b7a-44d9-bdc2-3b6366856dd6.png)
If you check the project requirements again, you will see that it says the name is to return a maximum of 10 characters. The extra `..` makes it 12 characters.